inst/tinytest/test_d_delayed_6.R

```diff
@@ -0,0 +1,54 @@
+# ================================================================
+# See comments on test_delayed_1.R regarding filenames for these test cases.
+# ================================================================
+
+# Local and non-local test files are in separate files. This is to (a) validate
+# correct functioning of non-cloud-dependent and cloud-dependent things; (b)
+# separate what can be tested without and with cloud credentials.  Let's keep
+# this separation of files.
+
+# ----------------------------------------------------------------
+library(tinytest)
+
+if ((unitTestToken <- Sys.getenv("TILEDB_REST_UNIT_TEST_TOKEN")) != "") {
+    Sys.setenv("TILEDB_REST_TOKEN"=unitTestToken)
+} else {
+    if (!file.exists("~/.tiledb/cloud.json")) exit_file("No authentication")
+}
+
+if ((namespaceToCharge <- Sys.getenv("TILEDB_REST_UNIT_TEST_NAMESPACE_TO_CHARGE")) == "") {
+    exit_file("No TILEDB_REST_UNIT_TEST_NAMESPACE_TO_CHARGE in environment")
+}
+
+# ----------------------------------------------------------------
+library(tiledbcloud)
+
+# ----------------------------------------------------------------
+a <- delayedSQL(
+    query="select `rows`, AVG(a) as avg_a from `tiledb://TileDB-Inc/quickstart_dense` GROUP BY `rows`",
+    name="rows-query",
+    namespace=namespaceToCharge)
+o <- compute(a, namespaceToCharge)
+expect_equal(names(o), c("avg_a", "rows"))
+expect_equal(rownames(o), c("1", "2", "3", "4"))
+expect_equal(o[["avg_a"]], c("2.5000", "6.5000", "10.5000", "14.5000"))
```

Comment:
Why are we getting floats back as strings here?

Reply:
@ihnorton I'll add a comment -- this is the way it works with SQL ... @Shelnutt2 may have more context than I ...

Reply:
Hmm, ok. Seems weird, but marking resolved if it's expected.

Reply:
This is because `AVG` actually returns a decimal type not a double/float type. Decimals are represented as strings to enforce the arbitrary precision. Most systems convert decimals to floats/doubles but sine we pipe straight to JSON for output I believe we get the string encodings.

https://mariadb.com/docs/reference/mdb/functions/AVG/

Reply:
Ok, no worries - thanks for the explanation.